ssh/tailssh/incubator.go

```diff
@@ -120,10 +121,18 @@ func (ss *sshSession) newIncubatorCommand() (cmd *exec.Cmd) {
 		if isShell {
 			incubatorArgs = append(incubatorArgs, "--shell")
 		}
-		if isShell || runtime.GOOS == "darwin" {
-			// Only the macOS version of the login command supports executing a
-			// command, all other versions only support launching a shell
-			// without taking any arguments.
+		// Only the macOS version of the login command supports executing a
+		// command, all other versions only support launching a shell
+		// without taking any arguments.
+		shouldUseLoginCmd := isShell || runtime.GOOS == "darwin"
+		if ipnlocal.CheckSELinux() {
```

Comment:
`if shouldUseLoginCmd && ipnlocal.CheckSELinux() {` ?

"Check" doesn't say much to me, though, bool-wise. Check what? I guess "whether we're using".

Maybe "UsingSELinux" ? Or "IsSELinuxEnforcing" ?